### PR TITLE
Update submodules

### PR DESCRIPTION
**Pull request by angrybayblade**

### CODE REVIEW

**Review by jmoreira-valory — COMMENTED** [review]
Misses update of hello-world repo ?

**Comment by jmoreira-valory** [comment]
**Please, each PO approve this PR after confirming that for the submodule linked to their repos:**

1. The submodule update is pointing to the intended commit, which should be the latest tag/release.

2. **The release workflow of the corresponding repository has finished successfully (green icon).**

3. To the best of your knowledge, the public-facing tutorial in the submodule (`<submodule_folder>/docs/index.md`) is up to date. In particular, you agree with the instructions in the section "Demo".

4. If a specific version of a dependency is required, it is made explicit in the section "Demo". In particular, the Open Autonomy version used by the toolkit should be explicit in the section "Demo".

5. You have replicated the instructions exactly as described in the section "Demo" (with the exact options described there: hash values, command line parameters, private keys, etc.), in a clean virtual environment, and it has worked. Also:
	a. The instructions therein are sufficient so that an external user is able to replicate the demo. **or**

	b. If this is not 100% possible, (e.g., due to requiring private API keys, registered addresses, etc) this limitation is clearly informed to the user in the tutorial.

**Comment by angrybayblade** [comment]
> Misses update of hello-world repo ?

I missed the hello world update since, it was not listed on the ticket I'll open a PR

**Review by 0xArdi — APPROVED** [review]

**Review by dvilelaf — APPROVED** [review]

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -52,4 +52,4 @@ jobs:
 
       - name: Generate Documentation
         run: |
-          poetry run mkdocs build --clean --strict
+          poetry run mkdocs build --clean
```
Comment (angrybayblade) [inline .github/workflows/docs.yml:55]: https://github.com/valory-xyz/docs/issues/72

File: HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+* MintKit: https://github.com/valory-xyz/agent-academy-1 - agent-academy-1 (v0.10.1)
+* CoordinationKit: https://github.com/valory-xyz/contribution-service - contribution-service (v0.6.1)
+* MLKit: https://github.com/valory-xyz/apy-oracle - apy-oracle (v0.10.13)
+* OracleKit: https://github.com/valory-xyz/price-oracle - price-oracle (v0.10.14)
+* IEKit: https://github.com/valory-xyz/IEKit - IEKit (v0.5.1)
+* GovKit: https://github.com/valory-xyz/governatooorr - governatooorr (v0.1.0.alpha13)
+* MechKit: https://github.com/valory-xyz/mech - mech (v0.5.2)
+* KeeperKit: https://github.com/valory-xyz/agent-academy-2 - agent-academy-2 (v0.6.1)
+* MessagingKit: https://github.com/valory-xyz/open-acn - open-acn
+
```
Comment (jmoreira-valory) [inline HISTORY.md:21]:
```

Demos
* Hello World: https://github.com/valory-xyz/hello-world.git - hello-world (v0.1.1)
```

Comment (jmoreira-valory) [inline HISTORY.md:21]:
```suggestion
```

Demos
* Hello World: https://github.com/valory-xyz/hello-world.git - hello-world (v0.1.1)
```
```

**Review by jmoreira-valory — APPROVED** [review]

**Review by Adamantios — APPROVED** [review]

**Pull request merged**